packages/ava/src/data/statistics/stdlib/constants.ts

```diff
@@ -0,0 +1,15 @@
+import { LOWESSOptions, PCorrTestOptions } from './types';
```

Comment:
do not use the name `stdlib`, these methods should be a part of `statistics`

Reply:
have change to 'algorithms'

Reply:
🤔 The algorithm should not be a subdirectory of statistics. In `stdlib`, it is `stdlib/stats/pcorrtest`

Reply:
have removed to data/statistics